### PR TITLE
feat(route): add anime1

### PR DESCRIPTION
**Pull request by cxheng315**
add 2 anime1 route:
1. anime
2. search

## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/anime1/anime/2024年夏季/神之塔-第二季
/anime1/search/神之塔
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
Contribution to a deprecated route: https://github.com/DIYgod/RSSHub/tree/master/lib/routes-deprecated/anime1

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Please remove deprecated files as well

File: lib/routes/anime1/search.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['anime1.me/?s=:keyword'],
```
Comment (TonyRL) [inline lib/routes/anime1/search.ts:26]: `source` can't match any search parameters

Comment (cxheng315) [reply lib/routes/anime1/search.ts:26]: Sorry, I don't quite follow this. The URL should look like `https://anime1.me/?s=123`, so the source should work

Comment (TonyRL) [reply lib/routes/anime1/search.ts:26]: `s` is the [search parameter](https://nodejs.org/api/url.html#urlsearchparams) which `source` can't match

Comment (cxheng315) [reply lib/routes/anime1/search.ts:26]: will `anime1.me/?s=${encodeURIComponent(:keyword)}` works?

Comment (TonyRL) [reply lib/routes/anime1/search.ts:26]: > `source` can't match any search parameters

No.

Comment (cxheng315) [reply lib/routes/anime1/search.ts:26]: any suggestion? or keep `anime1.me` only

Comment (TonyRL) [reply lib/routes/anime1/search.ts:26]: You can remove the rule.

**Pull request merged**